scripts/lib/mkcloud-common.sh

```diff
@@ -1,5 +1,16 @@
 # This file shares common code between mkcloud-${mkclouddriver}.sh files and qa_crowbarsetup.sh
 
+function to_ip
+{
+    safely getent hosts "$1" | head -n1 | awk '{print $1}' | grep .
```

Comment:
It's more than nitpicking ...

to 1.: sure that might happen, same as all other tools and commands that we call - they all can change at any point in time
to 2.: well, then you should not do `git grep "safely.*|"` ; and if it is safely that emits to STDOUT the to_ip function should not care, because safely will exit anyway

Even though these points contain valid improvements.